### PR TITLE
clarify watchCompletion only watches the stage, not the stream

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-stream/src/main/scala/akka/stream/javadsl/Queue.scala [hunk 1]
```diff
@@ -32,8 +32,8 @@ trait SourceQueue[T] {
   def offer(elem: T): CompletionStage[QueueOfferResult]
 
   /**
-   * Method returns a [[CompletionStage]] that will be completed if the stream completes,
-   * or will be failed when the operator faces an internal failure.
+   * Method returns a [[CompletionStage]] that will be completed if this stage completes,
```
Comment (johanandren) [inline akka-stream/src/main/scala/akka/stream/javadsl/Queue.scala]: I whink we are now using "operator" across the board so maybe here as well? (Also makes it less confusing with the completion stage)

**Review by johanandren — APPROVED** [review]
LGTM with one possible improvement.

**Comment by raboof** [comment]
#26214

**Comment by raboof** [comment]
PLS BUILD

**Review by patriknw — APPROVED** [review]
LGTM after s/stage/operator/

**Pull request merged**